### PR TITLE
Seb

### PR DESCRIPTION
**Pull request by TITAN9389**
Draft of UI with WebSocket (Redux/Mobx not included yet)

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "compilerOptions": {
+        "experimentalDecorators": true,
```
Comment (i0natan) [inline tsconfig.json]: @TITAN9389 Do we use TS somewhere? why TSConfig?

File: ui/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "scripts": {
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-app-rewired test",
+    "eject": "react-app-rewired eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": [
```
Comment (i0natan) [inline ui/package.json]: @TITAN9389 What is this? just looking to learn, appreciate if you can share a link to a tutorial on this (I'm a backend guy... :])

File: tsconfig.json (hunk 1 above)
Comment (TITAN9389) [reply tsconfig.json]: its for VSCode just - always throws error on Decorators.

File: ui/package.json (hunk 2 above)
Comment (TITAN9389) [reply ui/package.json]: the eslintConfig that comes with the CRA built in , or  you can include custom one for more Strict linting.
and browsers list is kind of prefix for css styles.

**Pull request merged**